README.md

```diff
@@ -574,15 +574,21 @@ To take control over the inner organization of each field (each form row), you c
 A field template is basically a React stateless component being passed field-related props so you can structure your form row as you like:
 
 ```jsx
+
+import {Help, ErrorList} from "react-jsonschema-form/lib/components/fields/SchemaField"; // You can import also the default 'Label' from SchemaField. Below we use a custom label.
```

Comment:
How about putting `Help` and `ErrorList` to their own files? There exists a `ErrorList` already in src/components, so this list could be named `InlineErrorList`.